node/README.md

```diff
@@ -85,6 +85,21 @@ const leftPad = require("left-pad");
 
 ## Contributing
 
+### Setting up the test runner
+
+This library contains automated tests pulled directly from the Node repo in
+order ensure compatibility.
+
+Setting up the test runner is as simple as running the `node/testdata/setup.ts`
+file, this will pull the configured tests in and then add them to the test
+workflow.
+
+To enable new tests, simply add a new entry inside `node/testdata/config.json`
+under the `tests` property. The structure this entries must have has to resemble
+a path inside `https://github.com/nodejs/node/tree/master/test`.
```

Comment:
Have you considered including a git submodule? Is it too big?

Reply:
A git submodule could suffice, but it makes it harder to patch some files that are needed for this to work, namely `common/index.js`.

That file is used in pretty much every single file, however it uses things like child_process that are not needed in most cases, so instead I just patch it and all the tests that are enabled continue to use it without any trouble

Not to mention this is more user friendly than submodules :)

Reply:
How about mocking `common/index.js` by injecting a fake module into `require.cache`, using a technique described [here](https://glebbahmutov.com/blog/put-mock-data-into-node-require-cache/)?

Reply:
That might be more complicated than the current solution though @kt3k